main/pipeline/pipeline.h

```diff
@@ -55,8 +55,9 @@ std::vector<ast::ParsedFile> autogenWriteCacheFile(const core::GlobalState &gs,
 // Note: `cancelable` and `preemption task manager` are only applicable to LSP.
 // If `intentionallyLeakASTs` is `true`, typecheck will leak the ASTs rather than pay the cost of deleting them
 // properly, which is a significant speedup on large codebases.
-void typecheck(const core::GlobalState &gs, std::vector<ast::ParsedFile> what, const options::Options &opts,
-               WorkerPool &workers, bool cancelable = false,
+void typecheck(const core::GlobalState &gs, std::vector<ast::ParsedFile> what,
+               std::vector<core::FileRef> extraFilesToFlush, const options::Options &opts, WorkerPool &workers,
```

Comment:
This should either be `std::vector<core::FileRef> &&extraFilesToFlush` or `const std::vector<core::FileRef> &extraFilesToFlush`